src/content/en/android/custom-tabs/implementation-guide/index.md

```diff
@@ -3,28 +3,29 @@ book_path: /web/android/_book.yaml
 description: Implementation Guide for Custom Tabs.
 
 {# wf_published_on: 2020-02-04 #}
-{# wf_updated_on: 2020-03-06 #}
+{# wf_updated_on: 2020-06-17 #}
 {# wf_blink_components: N/A #}
 
 # Custom Tabs Implementation guide {: .page-title }
 
-A complete example is available at [https://github.com/GoogleChrome/custom-tabs-client][1].It
+A complete example is available on [the GitHub sample application][1]. It
 contains re-usable classes to customize the UI, connect to the background service, and handle the
 lifecycle of both the application and the custom tab activity.
 
 If you follow the guidance from this page, you will be able to create a great integration.
 
-The first step for a Custom Tabs integration is adding the [Custom Tabs Support Library][2] to your
-project. Open your `build.gradle` file and add the support library to the dependency section.
+The first step for a Custom Tabs integration is adding the [AndroidX Browser Library][2] to your
+project. Open the `app/build.gradle` file and add the support library to the dependencies section.
```

Comment:
"support library" describes the old group id for this library.  Can this be changed to "browser library"?
```suggestion
project. Open the `app/build.gradle` file and add the browser library to the dependencies section.
```